js-miniapp-bridge/src/bridge.js

```diff
@@ -96,6 +97,21 @@ MiniAppBridge.getUniqueId = function() {
     );
   });
 };
+
+/**
+ * Associating requestPermission function to MiniAppBridge object
+ * @param {String} requestType Type of permission that is requested. For eg., location
+ */
+MiniAppBridge.requestPermission = function(requestType) {
+  return new Promise(function(resolve, reject) {
+    return MiniAppBridge.exec(
+      "requestPermission",
+      requestType,
```

Comment:
I'm thinking about changing `requestType` into `Int`. The code of `requestType` will be defined in android, ios and js sdk.
Example:
```
        const val GEOLOCATION = 1001
        const val LOCATION = 1002
```
@ts-leojoseph-r @corycaywood What do you think?

Reply:
We will end up managing/maintaining Int and String back and forth, which would create some confusion. I think it would be good to have a readable format

Reply:
Yeah, @ts-leojoseph-r  is right. Unfortunately I think using Int would be difficult to manage and sync across platforms.